src/tiled/abstractobjecttool.cpp

```diff
@@ -276,6 +288,24 @@ void AbstractObjectTool::showContextMenu(MapObjectItem *clickedObjectItem,
     Utils::setThemeIcon(removeAction, "edit-delete");
     Utils::setThemeIcon(propertiesAction, "document-properties");
 
+    QList<MapObjectItem*> underlyingObjects = listOfObjectItemsAt(scenePos);
+    if (underlyingObjects.size() > 1) {
+        menu.addSeparator();
+
+        QMenu *selectUnderlyingMenu = menu.addMenu(tr("Select an underlying object"));
```

Comment:
To separate object-related actions and a selection is a good idea. But where is it better to place it, in the abstract or in the selection tool? On the one hand, in the abstract it can be accessed from all tools, on the other, alt+click is working only with the selection one, so it will be more logical.